src/verso.rs

```diff
@@ -81,6 +81,8 @@ impl Verso {
     /// - Image Cache: Enabled
     pub fn new(evl: &ActiveEventLoop, proxy: EventLoopProxy<EventLoopProxyMessage>) -> Self {
         let config = Config::new();
+        let preferences = Preferences::default();
+        servo_config::prefs::set(preferences);
```

Comment:
Can we extract this to a function or put it in the Config init? in case it will grow up in the future

Reply:
Moving into config would probably make sense for now as we have yet to had our own Preference component and support for customizing Preference. I'll do that.